lightning/src/ln/msgs.rs

```diff
@@ -1499,10 +1461,7 @@ impl Readable for UnsignedChannelAnnouncement {
 	}
 }
 
-impl_writeable_len_match!(ChannelAnnouncement, {
-		{ ChannelAnnouncement { contents: UnsignedChannelAnnouncement {ref features, ref excess_data, ..}, .. },
-			2 + 32 + 8 + 4*33 + features.byte_count() + excess_data.len() + 4*64 }
-	}, {
+impl_writeable!(ChannelAnnouncement, {
```

Comment:
Why not `impl_writeable_msg`?

Reply:
It doesn't make sense, I think - the contained UnsignedChannelAnnouncement always reads until the end, so there can never be bytes left after reading it that fit in a TLV. Any TLV entries will be in the unsigned one.